### ISSUE TITLE
Setting to show participating notifications count on the panel

### ISSUE THREAD

**Issue by ava1ar** [freaktechnik/advanced-github-notifier#155]
Thanks for great extension! I am using it with GitHub Enterprise and having too much notifications in general, so I would prefer to be informed only about participating notifications and see only this number. Looks like it is easy to "hack" this since it just requires add parameter to the notifications request call, but making this configurable via options require little bit more work. Hopefully you can include this feature in one of the upcoming updates. Thanks!

**Comment by freaktechnik** [comment]
Yeah, it would essentially require a filter on the notification reason. I'd likely implement it as a run-time filter and not a filter at request time, so you can dynamically still show all notifications if you'd like to.
